src/main/java/cl/transbank/webpay/example/controller/webpay/WebpayPlusDeferredController.java

```diff
@@ -64,11 +64,16 @@ public ModelAndView create(HttpServletRequest request) {
     public ModelAndView commit(HttpServletRequest request) {
         String tokenWs = request.getParameter("token_ws");
         Map<String, Object> details = new HashMap<>();
-
-        if (tokenWs == null) {
+        String buyOrder = (String) request.getSession().getAttribute("buyOrder");
+        String sessionId = (String) request.getSession().getAttribute("sessionId");
+        if(request.getParameter("TBK_TOKEN")==null && tokenWs==null || request.getParameter("TBK_TOKEN").isEmpty() && tokenWs.isEmpty())
```

Comment:
The same here, move the if logic to a variable

Reply:
The suggested changes were made